server/game/cards/06-WoE/Plunder.js

```diff
@@ -0,0 +1,51 @@
+const _ = require('underscore');
+const Card = require('../../Card.js');
+
+class Plunder extends Card {
+    // Play: Reveal a random unrevealed card from your opponent's
+    // hand. Then, you may either repeat this effect or your opponent
+    // discards the last revealed card.
+    setupCardAbilities(ability) {
+        this.revealedCards = [];
```

Comment:
when is this array cleared?

Reply:
Why does it need to be cleared?  I modeled this on "One Stood Against Many", which never clears its array.

Reply:
Ohhh wait, yes it does, I see it now.  I'll figure out a fix.